lib/parallel_tests/cli.rb

```diff
@@ -35,9 +35,10 @@ def run_tests_in_parallel(num_processes, options)
 
       report_time_taken do
         groups = @runner.tests_in_groups(options[:files], num_processes, options)
+        groups.reject! &:empty?
 
         test_results = if options[:only_group]
-          groups_to_run = options[:only_group].collect{|i| groups[i - 1]}
+          groups_to_run = options[:only_group].collect{|i| groups[i - 1]}.compact
```

Comment:
If there's a better way I'm happy to fix. I think the current code works (at least the tests pass).